corehq/warnings.py

```diff
@@ -46,6 +48,7 @@
 
     # other, resolution not obvious
     ("IPython.core.interactiveshell", "install IPython inside the virtualenv.", UserWarning),
+    ("redis.connection", "distutils Version classes are deprecated. Use packaging.version instead."),
```

Comment:
Is this still needed? What code is importing them?

Reply:
yes, based on this [build failure](https://github.com/dimagi/commcare-hq/actions/runs/3886942464/jobs/6632632069), `module: redis.connection line 67`